assets/in

```diff
@@ -71,10 +72,15 @@ else
     fi
 fi
 
-if [ "$YARN_SUPPORT" = "true" ]; then
+if [ "$YARN_SUPPORT" != "false" ]; then
     NODE_ENV=development && yarn install >&2
 else
     NODE_ENV=development && npm install --quiet >&2
 fi
 
-mv node_modules "$DEST_DIR/"
+if [ "BOWER_SUPPORT" != "false" ]; then
```

Comment:
This should be $BOWER_SUPPORT instead of BOWER_SUPPORT. Otherwise this condition always executes regardless of the value of the $BOWER_SUPPORT value set on line 31.